src/test/java/com/automation/pages/activityStream/EventPage.java

```diff
@@ -7,16 +7,47 @@
 import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class EventPage extends AbstractPageBase {
+    // Tester Canan
 
     // Upload elements
     @FindBy (id = "bx-b-uploadfile-blogPostForm_calendar")
-    protected WebElement UploadFilesBtn;
+    protected WebElement uploadFilesIcon;
+
+    @FindBy (name = "bxu_files[]")
+    protected WebElement uploadFilesAndImages;
```

Comment:
This locator already exists, is it pointing to the same location as is already in the repository?
`   @FindBy(name = "bxu_files[]")	
    private WebElement uploadfilesandimages;`

Reply:
Yes, it is pointing the same. I did not recognize that it was already there.